SeleniumHQ/selenium
# Add SessionMap data

Partial-PR: #8020

### Testing Instructions
* Build the Grid by `bazel build selenium-server-standalone`
* Start the Selenium Grid by `java -jar selenium_server_deploy.jar standalone`
* Start querying using curl commands like : `curl -X POST -H "Content-Type: application/json" --data '{"query": "{ grid { nodes { id, uri, status, maxSession, capabilities, sessions {id, capabilities, startTime} }, uri, totalSlots, usedSlots, sessionCount } }"}' -s http://localhost:4444/graphql`
* Open new sessions using a Java file opening RemoteDrivers
* Start querying again using curl commands and notice the difference in the queries due to new sessions.
### Types of changes


### Checklist


- [x] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md) document.
